web/server/watch/integration_test.go

```diff
@@ -22,13 +21,13 @@ func TestWatcher(t *testing.T) {
 	}
 
 	log.SetFlags(log.LstdFlags | log.Lshortfile | log.Lmicroseconds)
-	output := new(bytes.Buffer)
```

Comment:
the name "output" was used again in another block in this file for a totally different variable with a different type